core/workspace.js

```diff
@@ -24,6 +24,7 @@
  */
 'use strict';
 
+goog.require('Blockly.VariableMap');
```

Comment:
doh. Sorry. The require should be after the provide... I think our eslint rules enforce this... but I'm not certain.